src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/admissioncontrol/AdmissionControlRca.java

```diff
@@ -73,50 +70,37 @@
     private final RangeConfiguration requestSizeHeapRange;
     private final int rcaPeriod;
     private int counter;
-    private double previousHeapPercent;
 
     public <M extends Metric> AdmissionControlRca(
             final int rcaPeriodInSeconds, final M heapUsedValue, final M heapMaxValue) {
         super(rcaPeriodInSeconds);
         this.counter = 0;
-        this.previousHeapPercent = 0.0;
         this.rcaPeriod = rcaPeriodInSeconds;
         this.heapUsedValue = heapUsedValue;
         this.heapMaxValue = heapMaxValue;
         this.requestSizeHeapRange = new RequestSizeHeapRangeConfiguration();
     }
 
-    private <M extends Metric> double getMetric(
-            M metric, Field<String> field, String fieldName, String dataField) {
-        AtomicReference<Double> metricValue = new AtomicReference<>((double) 0);
-        metric.getFlowUnits().stream()
-                .filter(flowUnit -> !flowUnit.isEmpty() && !flowUnit.getData().isEmpty())
-                .mapToDouble(
-                        flowUnit ->
-                                readDataFromSqlResult(
-                                        flowUnit.getData(), field, fieldName, dataField))
-                .forEach(
-                        metricResponse -> {
-                            if (Double.isNaN(metricResponse)) {
-                                LOG.debug(
-                                        "[AdmissionControl] Failed to parse metric from {}",
-                                        metric.name());
-                            } else {
-                                metricValue.set(metricResponse);
-                            }
-                        });
-        return metricValue.get();
+    private <M extends Metric> double getMetric(M metric, Field<String> field, String fieldName) {
+        double response = 0;
+        for (MetricFlowUnit flowUnit : metric.getFlowUnits()) {
```

Comment:
Can we run as ```./gradlew build ``` as it will fix spotless bugs if any.

Reply:
Done